src/styles/blocks/header.scss

```diff
@@ -0,0 +1,73 @@
+$sizes:
+  "1" 320px,
+  "2" 744px;
+
+.header {
+  display: grid;
+  gap: 24px;
+
+  padding: 0 120px;
+
+  grid-template-columns: repeat(12, 63px);
+
+  position: relative;
+  
+  height: 630px;
+  width: 1260px;
```

Comment:
do not control header with and height with hard coded values
for height use `vh`
let your content , padding , margin decide your width

Reply:
But my header has no content to fill, only one background image and two headings. 
Can I use a grid design that defines the width and height? 
Or should I wrap it all in a "page" container and the width of the "body defines the width of the header?